zilliqa/src/db.rs

```diff
@@ -253,6 +253,8 @@ impl Db {
                 PRIMARY KEY (address, tx_hash));
             CREATE TABLE IF NOT EXISTS tip_info (
                 latest_finalized_view INTEGER,
+                latest_view INTEGER,
+                latest_view_timestamp BLOB,
```

Comment:
nit: I quite like the convention of suffixing timestamp DB columns with `_at` - i.e. `latest_view_updated_at`?

Reply:
SQLite has a `TIMESTAMP` data type - Should we use it?

Reply:
Updated column name to `view_updated_at`. I'm not sure why weve used BLOB rather than timestamp but it does allow for simpler conversion to/from. We could move over to TIMESTAMP but i see no advantage to doing so?